src/web/services/__snapshots__/student.service.spec.ts.snap

```diff
@@ -0,0 +1,33 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`StudentService should generate course student list with section as csv 1`] = `
+ZoneAwarePromise {
```

Comment:
Can we just use the plain string?

Reply:
I don't know how to extract it from the promise :")

Reply:
A simple Google search will save you :P https://fireship.io/snippets/testing-rxjs-observables-with-jest/

Reply:
Today I learned 😂 thank you!

Reply:
Using `done()` is a general technique to test async method so you don't even need to convert to `Promise`

Reply:
Out of curiosity, why do you even need to bring `Promise` to the picture? The example above is literally what the production code is doing.

Reply:
I didn't know how to test for observable values :")